docs/eventing/features/transport-encryption.md

```diff
@@ -67,6 +156,10 @@ The possible values for `transport-encryption` are:
     - Addressables must not accept events to non-HTTPS endpoints
     - Addressables must only advertise HTTPS endpoints
 
+!!! important
+    The `strict` is only enforced on the Broker and Channel receiver/ingress. Senders can still send
+    events over plaintext HTTP.
```

Comment:
@pierDipi could we reword this? I find it a little confusing. To me, `Senders can still send events over plaintext HTTP` sounds like I can still send my events over http to a broker. But, my understanding is that we are trying to say that when a broker or channel sends events to a subscriber, if that subscriber only has a HTTP address, the broker or channel will send the event with HTTP instead of HTTPS.